maxMuscleActivation: [4.283463046781365e-05, 4.283463046781365e-05, 5.487194978307346e-05, 2.1171711157038163e-05, 3.926280981042861e-05, 5.881869953050275e-05, 5.982470247978585e-05, 4.094358637980476e-05]
minMuscleActivation: [5.452733017956047e-05, 3.452733017956047e-05, 2.565087245794032e-06, 2.036688765646669e-05, 1.6263800384786344e-05, 1.2486896710856919e-05, 5.9090420854171376e-05, 4.009358217706772e-05]
